src/analysis/exploration/findBlockedReaction.m

```diff
@@ -33,7 +33,7 @@
     end
 else
     solution = solveCobraLPCPLEX(model, 0, 0, 0, [], 1e-6);
-    blockedReactions = model.rxns(find(~solution.full))';
+    blockedReactions = model.rxns(find(abs(solution.full) < 1e-11))';

```

Comment:
shouldn't this be smaller than the feasibility tolerance, whatever that is? 
Also, you don't need the find any more if you have a logical array anyways.

Reply:
the test fails using the feasTol, and 1e-11 is more stringent in any case.